Homework5/BookTask/Catalog.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace BookTask
+{
+    internal class Catalog : IEnumerable<Book>
+    {
+        public List<Book> Books { get; set; }
+        public Catalog()
+        {
+            Books = new List<Book>();
+        }
+
+        public void AddBook(Book book)
+        {
+            Books.Add(book);
```

Comment:
The book should be added per ISBN where ISBN has to have format XXX-X-XX-XXXXXX-X or
XXXXXXXXXXXXX, where X is the number 0..9.
Current code accepts any ISBN.